File: src/targets/node/native/fixtures/short.js
<<<<<<< HEAD
const http = require('https');

const options = {
  method: 'GET',
  hostname: 'httpbin.org',
  port: null,
  path: '/anything',
=======
const http = require('http');

const options = {
  method: 'GET',
  hostname: 'mockbin.com',
  port: null,
  path: '/har',
>>>>>>> upstream/master
  headers: {}
};

const req = http.request(options, function (res) {
  const chunks = [];

  res.on('data', function (chunk) {
    chunks.push(chunk);
  });

  res.on('end', function () {
    const body = Buffer.concat(chunks);
    console.log(body.toString());
  });
});

req.end();

File: src/targets/ruby/native/fixtures/short.rb
require 'uri'
require 'net/http'
<<<<<<< HEAD
require 'openssl'

url = URI("https://httpbin.org/anything")

http = Net::HTTP.new(url.host, url.port)
http.use_ssl = true
=======

url = URI("http://httpbin.org/anything")

http = Net::HTTP.new(url.host, url.port)
>>>>>>> upstream/master

request = Net::HTTP::Get.new(url)

response = http.request(request)
puts response.read_body